include/Primitives/Plane.hpp

```diff
@@ -33,7 +36,7 @@ namespace Primitive {
              * @param axis Axis of the plane
              * @param position offset on axis
              */
-            Plane(Primitive::Axis axis, double position);
+            Plane(Primitive::Axis axis, double position, std::shared_ptr<Material::IMaterial> material);
```

Comment:
Describe parameter material in documentation

Reply:
it's from the merge of another branch